cristin-import/src/test/resources/features/JournalArticleRules.feature

```diff
@@ -1,30 +1,23 @@
-Feature: Mapping of "Article in business/trade/industry journal", "Academic article", "Popular scientific article", "Academic literature review" entries
+Feature: Mapping of "Article in business/trade/industry journal", "Academic article",
+  "Popular scientific article", "Academic literature review", "Short communication" entries
 
   Background:
     Given a valid Cristin Result with secondary category "ARTIKKEL_FAG"
 
-  Scenario: Cristin Result of type "Article in business/trade/industry journal" maps to "JournalArticle"
-    When the Cristin Result is converted to an NVA Resource
-    Then the NVA Resource has a Publication Instance of type "JournalArticle"
-    And the NVA JournalArticle Resource has a Content type of type "Professional article"
-
-  Scenario: Cristin Result of type "Academic article" maps to "JournalArticle"
-    Given a valid Cristin Result with secondary category "ARTIKKEL"
-    When the Cristin Result is converted to an NVA Resource
-    Then the NVA Resource has a Publication Instance of type "JournalArticle"
-    And the NVA JournalArticle Resource has a Content type of type "Research article"
 
-  Scenario: Cristin Result of type "Popular scientific article" maps to "JournalArticle"
-    Given a valid Cristin Result with secondary category "ARTIKKEL_POP"
+  Scenario Outline: Cristin Result of type "Academic chapter/article/Conference paper", "Chapter",
+  "Popular scientific chapter/article" and "Encyclopedia article" mapps to NVA entry type "ChapterArticle" and correct sub-type.
```

Comment:
Fix the title. probably listing all of the categories was not a good idea after all.